airflow/models/dagrun.py

```diff
@@ -724,7 +724,7 @@ def schedule_tis(self, schedulable_tis: Iterable[TI], session: Session = None) -
             ti
             for ti in schedulable_tis
             if (
-                ti.task.task_type == "DummyOperator"
+                (ti.task.task_type == "DummyOperator" or getattr(ti.task, "_is_dummy", False))
```

Comment:
```suggestion
                (getattr(ti.task, "_is_dummy", False)
```
Should this be enough? I think `DummyOperator` will also fall in this category

Reply:
This is just for backwards-compatibility so the current Serialized DAGs which don't have `_is_dummy` field still continue to work

Reply:
Maybe not needed after 67b5daf 🤔

Reply:
I think right now this could be used in Scheduler (on serialized dags) and Backfill (on real DAGs) so that commit alone won't help.